politeiawww/api/cms/v1/v1.go

```diff
@@ -539,15 +539,19 @@ type UserInvoicesReply struct {
 	Invoices []InvoiceRecord `json:"invoices"`
 }
 
-// AdminInvoices is used to get all invoices from all users
-type AdminInvoices struct {
-	Month  uint16         `json:"month"`  // Month of Invoice
-	Year   uint16         `json:"year"`   // Year of Invoice
-	Status InvoiceStatusT `json:"status"` // Current status of invoice
+// Invoices is used to get all invoices from all users (if no userid is
```

Comment:
If I read this correctly, the start and end time are for the submission timestamp, not the month that the invoice was billing for, correct?  If so, can we make that clear in this comment.